src/Cache.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Flex;
+
+use Composer\Cache as BaseCache;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class Cache extends BaseCache
+{
+    private static $lowestTags = [
```

Comment:
you can make this a constant. since php 5.6 array constants are supported.

Reply:
I don't see why a (public) constant would be better honestly...

Reply:
Yep, and constants visibility is only for PHP 7.1 so it wouldn't work for Flex which is compatible with 7.0